src/Forms/SearchableDropdownTrait.php

```diff
@@ -474,7 +474,7 @@ private function getDefaultSchemaValue()
         if (!$this->getIsLazyLoaded() && $this->hasMethod('getDefaultValue')) {
             return $this->getDefaultValue();
         }
-        return $this->Value();
+        return $this->getValue();
```

Comment:
This is called by `getSchemaStateDefaults()` - shouldn't that use `getFormattedValue()`?

Reply:
Updated